### DISCUSSION TITLE
App Development Requirements Question

### DISCUSSION THREAD

**Discussion by justcreateme** [microsoft/WindowsAppSDK#364] [Q&A] [answered]
Greetings all,

I apologize if my posting is incorrect as I am very new to this site and have no idea what I'm doing just yet. lol

Anyway I plan to create an adult entertainment app like never before. I have all these wonderful ideas in how the app will work and maintain itself but it's been a very long research path on thinking of where to even start to get this app going. 

I've been doing tons of research and I discovered that adult apps like the one I have in mind cannot be hosted through software like the app store on both IOS and Android. Therefore, I am trying to find out if I can make this adult app and host it on the internet to have it to where it can be installed onto mobile devices legally.

I want to know if this is possible.

Does anyone know more information on this or know where I can find more information regarding adult apps and their legalities? 

Please help if you can. 

Thanks so much!

**Comment by andrewleader** [comment, marked as answer]
Hi, thanks for asking, however this repository is for Windows desktop apps and not mobile apps. I would recommend searching for Android and iOS documentation/blogs/etc. From my limited knowledge, I believe Android will let you install apps from websites (bypassing the store), but iOS definitely does not (user's phones need to be jailbroken to install apps outside of the Store). But I can't help you any further, good luck.
